src/components/DictionaryList/DictionaryListPresentation.js

```diff
@@ -111,11 +110,13 @@ function DictionaryListPresentation({
                   </tr>
                 </thead>
                 <tbody className="bg-white divide-y divide-charcoal-200">
-                  {infiniteQueryResponse?.data?.pages?.map((page) => (
+                  {infiniteQueryResponse?.data?.pages?.map((page, j) => (
                     <Fragment key={page.pageNumber}>
-                      {page.results.map((entry) => (
+                      {page.results.map((entry, i) => (
                         <tr key={entry?.id}>
-                          <td className="px-6 py-4">
+                          <td
+                            className={`px-6 py-4 ${i === page.results.length - 1 && j === infiniteQueryResponse.data.pages.length - 1 ? 'pb-20' : ''}`}
```

Comment:
To reduce the repetition we could put this into a reusable variable 
i.e. `const ifLastEntryStyling = i === page.results.length - 1 && j === infiniteQueryResponse.data.pages.length - 1 ? 'pb-20' : ''`

Reply:
good idea!

Reply:
complete :)